src/parquet/encodings/plain-encoding-test.cc

```diff
@@ -141,7 +144,7 @@ void EncodeDecode<ByteArray, Type::BYTE_ARRAY>::generate_data() {
   int max_byte_array_len = 12 + sizeof(uint32_t);
   size_t nbytes = num_values_ * max_byte_array_len;
   data_buffer_.resize(nbytes);
-  random_byte_array(num_values_, 0.5, data_buffer_.data(), draws_,
+  random_byte_array(num_values_, 0, data_buffer_.data(), draws_,
```

Comment:
We are, but it was being lost in the CMakeLists.txt. I fixed it in PARQUET-457 https://github.com/apache/parquet-cpp/pull/58/files#diff-af3b638bc2a3e6c650974192a53c7291R238